### ISSUE TITLE
customize keymap and layout

### ISSUE THREAD

**Issue by goldengrape** [flit/MidiKeys#19]
I want to be able to define the key and tone mapping myself, can you make KeyMaps.plist customizable for editing? You can refer to VMPK https://vmpk.sourceforge.io/

In addition, the appearance of the piano is not really cool enough, and in some cases it is not the optimal arrangement according to the way the piano is arranged. It would be better if you could define it yourself. For example, like drum pad, see noise.app

**Comment by xoxo369** [comment]
I agree this is an issue and would really love to see this program updated with keymap ability AND the ability to save keymap loadouts, I too right away checked the keymaps.plist to see if I could edit it hehe, I think this program is the best I have found for Mac type to midi but it needs those features badly..

**Comment by xoxo369** [comment]
awesome! I mapped out every scale there is and I will upload my .plist to
the github soon, thank you so much for your help!! :)
